packages/core/lib/commands/develop/run.js

```diff
@@ -1,5 +1,6 @@
 const emoji = require("node-emoji");
 const mnemonicInfo = require("../../mnemonics/mnemonic");
+const configureGanacheOptions = require("../../configureGanacheOptions");
```

Comment:
Just a style thing but how would you feel about just destructuring the method we want to use? It looks a little bit cleaner...like
```
const { configureManagedGanache } = require(....
```